src/nidmm/metadata/functions_addon.py

```diff
@@ -12,7 +12,7 @@
     'CheckAttribute.+':                { 'codegen_method': 'no',       },  # We do not include any Check Attribute functions
     '.etAttribute.+':                  { 'codegen_method': 'private',  },  # All Set/Get Attribute functions are private
     'init':                            { 'codegen_method': 'no',       },
-    'error_message':                   { 'codegen_method': 'no',       },
+    'error_message':                   { 'codegen_method': 'private',  },
     'GetError':                        { 'codegen_method': 'private',  },
     'GetErrorMessage':                 { 'codegen_method': 'private',  },
```

Comment:
If these are now dead code, then don't generate.
Applies to all.

Reply:
fixed